src/main/java/org/mule/extensions/jms/internal/config/JmsConfig.java

```diff
@@ -51,7 +50,8 @@ public class JmsConfig
   @Inject
   SchedulerService schedulerService;
 
-  private Scheduler scheduler;
+  private static Scheduler scheduler;
```

Comment:
Are we gonna use the same Scheduler for all the instances ? we can not shout it down in the dispose ?? 🤔